src/trust_anchor_util.rs

```diff
@@ -16,25 +16,33 @@
 
 use std;
 use super::{Error, TrustAnchor};
-use super::cert::{EndEntityOrCA, parse_cert};
+use super::cert::{EndEntityOrCA, Cert, parse_cert};
+use super::cert::certificate_serial_number;
+use super::der;
 use untrusted;
 
+fn trust_anchor_from_cert<'a>(cert: Cert<'a>) -> TrustAnchor<'a> {
```

Comment:
Please put this function between `cert_der_as_trust_anchor` and `parse_cert_v1`. In general, we put public things ahead of the implementation details of the public thing, and we try to put things roughly in order that they are used, so that somebody wanting to read `cert_der_as_trust_anchor` should be able to grep for it and then scroll down.